src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -131,26 +131,26 @@ class CaseOutputDescription:
 @attr.s(kw_only=True)
 class _MassSourceCommon:
     """
-    :ivar total_mass_flow_rate:
+    :param total_mass_flow_rate:
```

Comment:
Was this change intentional to change the text in the documentation (from `Variable` to `Parameters`)? 

Because `:param:` is a well-known field for functions parameter, and perhaps we should stick with an explicit version `:parameter:`, so it will be clear that is this field is not related to a function parameter.

what are your thoughts on this @oliveira-mauricio @prusse-martin

Reply:
Not sure I follow... the original intent was to document the instance variables right?

I prefer `:param:` over `:parameter:` because the former is shorter and there's really no confusion when using it.

Reply:
I don't get the reason for changing this, what is the goal? Also, where is this in the online docs? I couldn't find it.

Reply:
Talking to @williamjamir I get more context about the documentation. 

It was my bad, as I saw `:param` being used in another class (i.e `PvtModelsDescription`) so I thought it would be great to have a pattern. Also using `:param` renders a `Parameters` section in the docs over a `Variables` (William mention in the comment above), witch I think is more intuitive for users that aren't exactly programmers.

Another point it that we have a lot of blank space using `:param` or `:ivar`: 
https://alfasim-sdk.readthedocs.io/en/latest/alfacase/api_reference.html?highlight=%20PvtModelsDescription#alfasim_sdk.PvtModelsDescription

I think we have a lot to debate about the docs, but for this PR I will stick to the plan of reviewing the links and removing types from docs.

Reply:
> I think we have a lot to debate about the docs, but for this PR I will stick to the plan of reviewing the links and removing types from docs.

I agree. 👍

Reply:
That means you will change those back to ivar?

Reply:
Yeah, I changed to `:ivar` but now `sphinx` can't find the reference for those: `WARNING: py:obj reference target not found: correlations`

Reply:
With `:param` it doesn't show any warning

Reply:
Just to register: this warning on :ivar: was fixed in a more recent version of sphinx, we will try to update it to solve this issue.